src/tokens.ts

```diff
@@ -22,7 +22,7 @@ export const MIME_TYPE = 'text/x-robotframework';
 export const MODE_NAME = 'robotframework';
 
 /** the human-readable name of the CodeMirror mode */
-export const MODE_LABEL = 'Robot Framework';
+export const MODE_LABEL = 'robotframework';
```

Comment:
:+1:

Reply:
I had a dejavu that I did had this same issue years ago :see_no_evil:

Reply:
this is actually better UI, too, as it doesn't hide the `resource` label:

![Screenshot from 2022-04-11 12-22-27](https://user-images.githubusercontent.com/45380/162795403-729acff3-4e0b-4ffc-b4a6-18f4ea87820e.png)

Reply:
with a fair amount of `pip install --pre`, all seems to work together:

![Screenshot from 2022-04-11 12-45-57](https://user-images.githubusercontent.com/45380/162798947-3036a83f-ef39-46f3-8b3e-ac47d1aeb1ec.png)